docs/content/config/advanced/mail-forwarding/gmail-smtp.md

```diff
@@ -0,0 +1,52 @@
+---
+title: 'Mail Forwarding | GMAIL SMTP'
+---
+
+[GMAIL SMTP (Simple Message Transport Protocol)][gmail-smtp] provides a simple way for cloud based applications to send and receive email.
+
+!!! example "Configuration via ENV"
+
+    [Configure a relay host in DMS][docs::relay] to forward all your mail through GMAIL SMTP:
+
+    - `RELAY_HOST` should match your [SMTP Server Endpoint][gmail-smtp::relay-host].
+    - `RELAY_PORT` should be set to [one of the supported Gmail SMTP ports][gmail-smtp::relay-port] (_eg: 587 for STARTTLS_).
+    - `RELAY_USER` and `RELAY_PASSWORD` should be set to your [Gmail Account ID][gmail-smtp::account-id].
+
+    ```env
+    RELAY_HOST=smtp.gmail.com
+    RELAY_PORT=587
+    # Alternative to RELAY_HOST + RELAY_PORT which is compatible with LDAP:
+    DEFAULT_RELAY_HOST=[smtp.gmail.com]:587
+
+    RELAY_USER=someone@gmail.com
+    RELAY_PASSWORD=xxx
+    ```
```

Comment:
There are [two ways to configure for relay with gmail](https://support.google.com/a/answer/176600)
- Your [initial link](https://support.google.com/a/answer/2956491#zippy=%2Cpostfix) points to the first option which is via `smtp-relay.gmail.com` but requires setup through [Google Workspace](https://workspace.google.com) and trusts via IP address.
- The 2nd option is for `smtp.gmail.com` and can be either via Google Workspace account or your personal gmail. This seems to require you to have already signed in, especially if it's the users regular credentials and there's 2FA involved, they'd perhaps use this on a personal system with a mail client like Thunderbird.

So it seems that the first option is better to advise users configuring DMS on a headless server? 🤔 

```suggestion
    - `RELAY_HOST` should be either `smtp.gmail.com` (_for a personal GMAIL account_) or `smtp-relay.gmail.com` (_when using Google Workspace_). For more information, view [these docs for the two supported SMTP endpoints][gmail-smtp::relay-host].
    - `RELAY_PORT` should be set to [one of the supported Gmail SMTP ports][gmail-smtp::relay-port] (_eg: 587 for STARTTLS_).
    - `RELAY_USER` and `RELAY_PASSWORD` should be set to your credentials for [Gmail][gmail-smtp::account-id].

    ```env
    RELAY_HOST=smtp.gmail.com
    RELAY_PORT=587
    # Alternative to RELAY_HOST + RELAY_PORT which is compatible with LDAP:
    DEFAULT_RELAY_HOST=[smtp.gmail.com]:587

    RELAY_USER=username@gmail.com
    RELAY_PASSWORD=secret
    ```
```

- I don't have time to reproduce so I'll go with your personal Gmail account example provided.
- The advice to set `RELAY_USER` + `RELAY_PASSWORD` to "Gmail Account ID" seems vague?
  - If this is your normal username and password it would be the credentials, similar to the equivalent line in AWS SES relay host docs page you've adapted for this PR.
  - I don't think you're meant to set both of these ENV to the same value. Your example also shows they're different.
   - > ![image](https://github.com/docker-mailserver/docker-mailserver/assets/5098581/b2550734-7467-46d3-ad6a-d663fd6fa821)

For the `smtp-relay.gmail.com` option, it seems authentication is either IP or credentials based (Google Workspace):

> ![image](https://github.com/docker-mailserver/docker-mailserver/assets/5098581/f72ac3b4-7d55-485e-9a13-1ff0663bdda4)

---

References:

> ![image](https://github.com/docker-mailserver/docker-mailserver/assets/5098581/6e208081-89f2-4860-9907-5bf63db1785b)

> ![image](https://github.com/docker-mailserver/docker-mailserver/assets/5098581/6710df1d-a2b4-42e3-9c65-742cfef88411)

> ![image](https://github.com/docker-mailserver/docker-mailserver/assets/5098581/a2b98d6b-28c5-4bf8-881a-77c05e40405e)

Reply:
your comment is perfectly correct with my opinion that i can't explain it. Thank you!!!!